### PR TITLE
Add Sokoban tutorial

### PR DESCRIPTION
**Pull request by cwalther**
Here is a first draft for the Sokoban tutorial. It may have mistakes, and I will probably find wording improvements after letting it rest for a while, but we can try it as it is tomorrow.

The evolution of the code follows https://github.com/cwalther/pewpew-tutorial-sokoban/commits/master, but I consider that a temporary repository and don’t think there is a need to keep it around. I would suggest, however, updating https://github.com/pewpew-game/game-sokoban to match the result of the tutorial (or the other way around), so that we don’t have two slightly different versions of Sokoban floating around.

### CODE REVIEW

File: docs/tutorial-sokoban.rst [hunk 1]
```diff
@@ -0,0 +1,401 @@
+Sokoban Tutorial
+****************
+
+When looking for simple games to implement on PewPew, it helps to look back to
+the times when computers were so young and underpowered, compared to today, that
+simple games were all they could run. One of those games is Sokoban. Let’s try
+to write a Sokoban for PewPew!
+
+Sokoban, which you can play in your browser at https://www.sokobanonline.com/,
```
Comment (deshipu) [inline docs/tutorial-sokoban.rst]: I'm not sure we want to advertise that website. Maybe we could link to https://en.wikipedia.org/wiki/Sokoban instead?

Comment (cwalther) [reply docs/tutorial-sokoban.rst]: Good point, I didn’t spend much thought on that. Fixed.

File: docs/tutorial-sokoban.rst [hunk 2]
```diff
@@ -0,0 +1,401 @@
...
+        pew.show(screen)
+        pew.tick(1/6)
+
+Let’s add a player character, indicated by a bright pixel, that can be moved
+around with the directional keys. We erase the pixel in the old position at the
+beginning of the loop by replacing it with 0 (black floor), then move the
+position if any keys are pressed, and finally paint it at the new position:
+
+.. code-block:: python
+    :emphasize-lines: 5,6,9,11-19
```
Comment (deshipu) [inline docs/tutorial-sokoban.rst:47]: nic, I didn't know this existed!

**Review by deshipu — APPROVED** [review]

File: docs/tutorial-sokoban.rst (hunk 2 above)
Comment (cwalther) [reply docs/tutorial-sokoban.rst:47]: Neither did I, but I thought it would be useful so I went looking for it in the Sphinx docs.

**Comment by cwalther** [comment]
It doesn’t seem to show up on pewpew.readthedocs.io, do we need to do anything special to start a build? Or is that because it’s still pointing at deshipu/pewpew and moving that to pewpew-game/pewpew broke it?

**Comment by deshipu** [comment]
I'm trying to get this fixed, basically for some reason that account on read the docs is disabled and I can't trigger builds: https://github.com/readthedocs/readthedocs.org/issues/6128

If I can't get that resolved, I will move it all to my own hosting.

**Comment by deshipu** [comment]
It's fixed now, and the new tutorial is online. I also updated the ``game-sokoban`` repository. Thank you!

**Comment by cwalther** [comment]
Thanks!

**Pull request merged**